server/handlers/assessment.js

```diff
@@ -0,0 +1,64 @@
+const AWS = require('aws-sdk');
+var ep = new AWS.Endpoint(process.env.DYNAMO_ENDPOINT);
+const dynamoDB = new AWS.DynamoDB({region: 'eu-west-2', endpoint: ep});
+
+async function add(request, h) {
```

Comment:
What is the `h` doing here? might want to be a little more descriptive with the name?

Reply:
It's the new hapi [response toolkit](https://github.com/hapijs/hapi/blob/master/API.md#response-toolkit) they added in v17. `h` is what they call it, so I just went with that

Reply:
Ohh! I've not looked at hapi in a while. 

:) 

Thanks for that!